cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -206,10 +206,10 @@
     "accelerator": "DESY-PETRA",
     "authors": [
       {
-        "name": "Bowdery"
+        "name": "Bowdery, C.K."
       },
       {
-        "name": "Pryce"
+        "name": "Pryce, J.J."
```

Comment:
In case of two initials, we should use space and leave out the dots, e.g. "Pryce, J J".

Can you please amend this here and elsewhere for people having two initials?